src/Compilers/CSharp/Test/Semantic/Diagnostics/OperationAnalyzerTests.cs

```diff
@@ -1185,6 +1185,37 @@ class C
         }
 
         [Fact]
+        public void OwningSymbolCSharp()
+        {
+            const string source = @"
+class C
+{
+    public void UnFunkyMethod()
+    {
+        int x = 0;
+        int y = x;
+    }
+
+    public void FunkyMethod()
+    {
+        int x = 0;
+        int y = x;
+    }
+
+    public int FunkyField = 12;
+    public int UnFunkyField = 12;
```

Comment:
"OwningSymbol" seems to make sense for context objects analyzing entire code blocks or operation blocks. Probably "ContainingSymbol" seems more intuitive for syntax node and operation analysis contexts?